test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/test.policy

```diff
@@ -1,3 +1,23 @@
+//  Copyright (c) 2019, 2019 IBM Corp. and others
```

Comment:
> The following files were modified and have incorrect copyrights
> test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/test.policy

Not sure how to address this, @fjeremic there's no line in the file, so our scripts don't know what to do.

I've copied this blob from:
https://github.com/eclipse/openj9/blob/master/test/functional/Java8andUp/testacc.policy#L1-L19
(with the year changed to 2019)

Reply:
The file didn't have a copyright, it doesn't need to be added. We can ignore the copyright failure.

Reply:
Ok. I'll drop that commit